ada/ada

```diff
@@ -1734,7 +1754,16 @@ validate_input() {
         echo 1>&2 "ERROR: no tokenfile, nor variable BEARER_TOKEN specified."
         exit 1
       fi
-      check_token "$token" || exit 1
+      # Token should be valid - unless we want to view the token!
+      # We should be able to view also invalid tokens (to check what's wrong with them).
+      if [[ $command != 'viewtoken' ]] ; then
+        if $recursive ; then
+          # For recursive operations, we want the token to be valid for at least 60 seconds.
+          check_token "$token" "$token_debug_info" 60 || exit 1
```

Comment:
It would be more clear if the `min_valid_time` is the same for recursive and normal operations. Couldn't we just use 60 seconds for both?

Reply:
Fix in commit b7ce7992b70e94ba255694041fdef13ec4c170d2